x-pack/plugins/aiops/public/components/explain_log_rate_spikes/explain_log_rate_spikes_analysis.tsx

```diff
@@ -228,16 +237,31 @@ export const ExplainLogRateSpikesAnalysis: FC<ExplainLogRateSpikesAnalysisProps>
         </>
       ) : null}
       {showSpikeAnalysisTable && foundGroups && (
-        <EuiFormRow display="columnCompressedSwitch" label={groupResultsMessage}>
-          <EuiSwitch
-            data-test-subj={`aiopsExplainLogRateSpikesGroupSwitch${groupResults ? ' checked' : ''}`}
-            showLabel={false}
-            label={''}
-            checked={groupResults}
-            onChange={onSwitchToggle}
-            compressed
-          />
-        </EuiFormRow>
+        <>
+          <EuiFlexGroup direction="column" gutterSize="none">
+            <EuiFlexItem>
+              <EuiFormRow display="columnCompressedSwitch" label={groupResultsMessage}>
+                <EuiSwitch
+                  data-test-subj={`aiopsExplainLogRateSpikesGroupSwitch${
+                    groupResults ? ' checked' : ''
+                  }`}
+                  showLabel={false}
+                  label={''}
+                  checked={groupResults}
+                  onChange={onSwitchToggle}
+                  compressed
+                />
+              </EuiFormRow>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              {groupResults && (
+                <EuiFormRow helpText={groupResultsHelpMessage}>
```

Comment:
I'd be tempted to set `fullWidth={true}` on the `EuiFormRow` so that the help message doesn't need to wrap onto a second line.